modules/iam-role-for-service-accounts-eks/variables.tf

```diff
@@ -351,6 +351,12 @@ variable "attach_vpc_cni_policy" {
   default     = false
 }
 
+variable "vpc_cni_enable_cloudwatch_logs" {
+  description = "Enable CloudWatch Logs policy for VPC CNI"
```

Comment:
Could we add a bit more detail around the use of the log group being primarily for network policy agent flow log support